### DISCUSSION TITLE
posts/a-way-to-prevent-all-attacks/

### DISCUSSION THREAD

**Discussion by giscus** [kristi-balla/kristi-balla.github.io#2] [General]
# posts/a-way-to-prevent-all-attacks/

Cyber attacks are a growing concern for individuals and organizations alike. According to a recent report by Statista, ransomware was the most frequently detected cyberattack worldwide in 2022, accounting for around 68% of all detected cyberattacks. Network breaches ranked second, with over 18% of the detections. Although less frequently, data exfiltration was also among the detected cyber attacks. [1]

http://127.0.0.1:4000/posts/a-way-to-prevent-all-attacks/

**Comment by kristi-balla** [comment]
I was wondering whether the way I referenced was the right one. I am aware of the fact that this isn't academic, but I'm still interested in your opinion. How would you have referenced?

**Reply by s6dalaue** [reply]
Is hyperref available, ich mein ja die referenzen stehen da but you cant click on them and have to scroll down to finde the specific one

**Reply by kristi-balla** [reply]
no, but markdown has a solution for that, already on it :thumbsup:

**Comment by s6dalaue** [comment]
Is hyperref available, ich mein ja die referenzen stehen da but you cant click on them and have to scroll down to finde the specific one

**Reply by s6dalaue** [reply]
and a question: can you delete comments?

**Reply by kristi-balla** [reply]
yes, you can, just not directly on the website. Comments are powered by giscus in the form of GitHub discussions, so basically you have to go [here](https://github.com/kristi-balla/kristi-balla.github.io/discussions/2) and delete your comment ;)
